doc/reference/algorithms/centrality.rst

```diff
@@ -45,8 +45,9 @@ Current Flow Closeness
    :toctree: generated/
 
    betweenness_centrality
-   edge_betweenness_centrality
+   betweenness_centrality_source
```

Comment:
This one has an `# obsolete name` comment in the source code. Perhaps it should be officially deprecated? I had thought that maybe this was a case of "deprecation-by-docs" i.e. lightly discourage users from using functions by making them harder to discover :)

(Note: this same comment also applies to `edge_betweenness`, which is also missing from the docs)

Reply:
Yes, I think both of these old names should really be deprecated.